Sources/PubNub/APIs/Objects+PubNub.swift

```diff
@@ -90,18 +104,20 @@ public extension PubNub {
       self.customFields = customFields
       self.channelFields = channelFields
       self.statusField = statusField
+      self.typeField = typeField
       self.channelCustomFields = channelCustomFields
       self.channelTypeField = channelTypeField
       self.channelStatusField = channelStatusField
       self.totalCount = totalCount
     }
 
-    var customIncludes: [ObjectsMembershipsRouter.Include]? {
+    var includeFields: [ObjectsMembershipsRouter.Include]? {
```

Comment:
So it had a wrong name for all this time?

Reply:
Yeah, I would say the previous name was quite confusing